substrate/frame/revive/src/exec.rs

```diff
@@ -2129,6 +2147,25 @@ where
 	fn last_frame_output_mut(&mut self) -> &mut ExecReturnValue {
 		&mut self.top_frame_mut().last_frame_output
 	}
+
+	fn copy_code_slice(&mut self, buf: &mut [u8], address: &H160, code_offset: usize, len: usize) {
```

Comment:
The len is already implicitly given by the `buf`.

```suggestion
	fn copy_code_slice(&mut self, buf: &mut [u8], address: &H160, code_offset: usize) {
```